cloudify_cli/commands/ssh.py

```diff
@@ -20,6 +20,8 @@
 import platform
 from distutils import spawn
 import re
+import subprocess
+import os
 
```

Comment:
I like to sort the imports by length, but it's not critical...
import os
import re
import platform
...
from distutils import spawn